zou/utils/movie.py

```diff
@@ -36,9 +36,21 @@ def generate_thumbnail(movie_path):
     file_target_name = "%s.png" % file_source_name[:-4]
     file_target_path = os.path.join(folder_path, file_target_name)
 
-    ffmpeg.input(movie_path, ss="00:00:00").output(
-        file_target_path, vframes=1
-    ).run(quiet=True)
+    try:
+        ffmpeg.input(movie_path, ss="00:00:00").output(
+            file_target_path, vframes=1
+        ).run(quiet=True)
+    except ffmpeg._run.Error as e:
```

Comment:
The code that handles the error is similar every time, can you make a helper function to display and raise the error?